crates/payload/builder/src/payload.rs

```diff
@@ -137,6 +142,15 @@ pub struct PayloadBuilderAttributes {
     pub withdrawals: Vec<Withdrawal>,
     /// Root of the parent beacon block
     pub parent_beacon_block_root: Option<B256>,
+    /// NoTxPool option for the generated payload
+    #[cfg(feature = "optimism")]
+    pub no_tx_pool: bool,
+    /// Transactions for the generated payload
+    #[cfg(feature = "optimism")]
+    pub transactions: Vec<TransactionSigned>,
+    /// The gas limit for the generated payload
+    #[cfg(feature = "optimism")]
+    pub gas_limit: Option<u64>,
```

Comment:
same pattern

Reply:
Addressed in https://github.com/anton-rs/op-reth/pull/128